### PR TITLE
Open graph improments

### PR DESCRIPTION
**Pull request by cytadela8**
- Better opengraph image 
- Improve Opengraph meta props 

This changes are live https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwarsztatywww.pl

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -11,18 +11,21 @@
...
-    <meta property="og:image" content="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% static 'images/black.png' %}">
+    <meta property="og:url" content="https://warsztatywww.pl{{ request.path }}" />
+    <meta property="og:locale" content="PL_pl" />
+    <meta property="og:type" content="website" />
+    <meta property="fb:app_id" content="1522345984699744" />
+    <meta property="og:image" content="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% static 'images/fb_image-cut.jpg' %}">
     <meta property="og:description"
-          content="Wakacyjne Warsztaty Wielodyscyplinarne to coroczna impreza organizowana przez studentów,
+          content="WWW to coroczna impreza organizowana przez studentów,
             przeznaczona dla licealistów zainteresowanych matematyką, informatyką, fizyką lub astronomią.">
```
Comment (krzys-h) [inline templates/base.html:21]: We added biology on the main page, how about here?

File: templates/base.html [hunk 2]
```diff
@@ -11,18 +11,21 @@
...
-    <title>
-      {% if title %}
-        {{ title }} - WWW
-      {% else %}
-        Wakacyjne Warsztaty Wielodyscyplinarne
-      {% endif %}
-    </title>
+    {% if title %}
+    <title>{{ title }} - WWW</title>
+    <meta property="og:title" content="{{ title }} - WWW" />
```
Comment (krzys-h) [inline templates/base.html:24]: Do you want links to the program page to show up as `Program WWW16 - WWW` (without the full name, but with WWW twice?)

Comment (cytadela8) [reply templates/base.html:24]: Hmm, maybe let's remove `- WWW` completelly?

File: templates/base.html (hunk 1 above)
Comment (cytadela8) [reply templates/base.html:21]: nah

File: templates/base.html (hunk 2 above)
Comment (krzys-h) [reply templates/base.html:24]: In that case, workshop pages will be just the workshop title without mention of WWW. The program page definitely doesn't need it though...

**Pull request merged**